util/postman/openkit.postman_environment.json

```diff
@@ -0,0 +1,123 @@
+{
+	"id": "b4c4d4a9-7480-4823-98a2-60bdf5b7f4fe",
+	"name": "openkit - prod",
+	"values": [
+		{
+			"key": "openkit_auth_url",
+			"value": "https://auth.prod.latch.com",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "openkit_url",
+			"value": "https://rest.latchaccess.com/access/sdk",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "openkit_passwordless_client_id",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "openkit_passwordless_client_secret",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "openkit_m2m_client_id",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "openkit_m2m_client_secret",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "openkit_user_token",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "openkit_user_refresh_token",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "openkit_partner_token",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "openkit_partner_refresh_token",
+			"value": "",
+			"type": "secret",
+			"enabled": true
+		},
+		{
+			"key": "param_user_email",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_user_uuid",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_door_uuid",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_otp",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_first_name",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_last_name",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_email",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_start_time",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "param_end_time",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		}
```

Comment:
Do we want to include the `passcodeType` param as well here?

Reply:
The current APIs only support `PERMANENT` passcode, so it's hard-coded in the request. Once we expose the other endpoints, we can add it.